### DISCUSSION TITLE
How to export Linux VM as a template

### DISCUSSION THREAD

**Discussion by jack99trade** [apache/cloudstack#9724] [Q&A] [answered]
Hi All,  I was looking for the option to export Linux VM as a template but the documentation link is broken 

"You are now ready to create the Final Template, for more information see [“Creating a Template from an Existing Virtual Machine”](https://docs.cloudstack.apache.org/en/4.19.1.1/adminguide/templates/_create_linux.html#creating-a-template-from-an-existing-instance)."   .

any suggestions how do I find this option?

**Comment by weizhouapache** [comment, marked as answer]
the link is incorrect
refer to https://docs.cloudstack.apache.org/en/4.19.1.1/adminguide/templates.html#creating-a-template-from-an-existing-instance

**Reply by jack99trade** [reply]
@weizhouapache  Thank you this works. 

At the bottom of this page https://docs.cloudstack.apache.org/en/4.19.1.1/adminguide/templates/_create_linux.html    the URL need to be updated to correct one.
